mappings/net/minecraft/inventory/CraftingInventory.mapping

```diff
@@ -1,4 +1,14 @@
-CLASS net/minecraft/unmapped/C_fcvwjvlq net/minecraft/inventory/CraftingInventory
-	METHOD m_dvbyvcly getHeight ()I
-	METHOD m_mmnfalel getIngredients ()Ljava/util/List;
-	METHOD m_rkeovwln getWidth ()I
+CLASS net/minecraft/unmapped/C_ektmmzcp net/minecraft/inventory/CraftingInventory
+	FIELD f_dxgcuxkd height I
+	FIELD f_wjzlqjky handler Lnet/minecraft/unmapped/C_mkrkudpa;
+	FIELD f_xtifskty width I
+	FIELD f_ypsnrmaz stacks Lnet/minecraft/unmapped/C_rnrfftze;
```

Comment:
why `stacks` instead of `ingredients`?

Reply:
The diff doesn't show it well but it's not exactly the same class anymore, it's `C_ektmmzcp` which already had those mappings.